src/app/carbon-estimator-form/carbon-estimator-form.component.html

```diff
@@ -9,11 +9,12 @@
         </label>
         <input
           id="headCount"
+          #headCountInput
           class="tce-border tce-border-slate-400 tce-rounded"
           formControlName="headCount"
           name="headCount"
           type="number"
-          min="1"
+          required
```

Comment:
I moved the min validation logic to component.ts so it wasn't split between component.html and component.ts. The [Angular docs](recommend) adding the required field to the html for accessibility reasons  even if it isn't required for validation because Validators.required is already being used.